### DISCUSSION TITLE
How to recenter the map on button click

### DISCUSSION THREAD

**Discussion by kingsleyfrancis** [vue-leaflet/vue-leaflet#353] [Q&A]
I am trying to recenter the map to a new [lat, lon] when a button is clicked, but I can't somehow seem to. I can't find any sample or documentation around recentering. What I have tried.

- I have checked `this.$refs.map.leafletObject` non of the methods seems to have anything related to recentering the map. 
- I have tried assigning directly to the center property `this.center = [lat, lon]`. But this only works on initialization not on assignment when a click event happens. Assigning to zoom works on click event.

Any help pointing me to the right direction is much welcomed. I'm using `"@vue-leaflet/vue-leaflet": "^0.10.1"`

**Comment by Amarildoo** [comment]
I managed to re-center the map using:
```html
<l-map :zoom="map.zoom" :center="map.center" @click="updateMarkerPosition">...
```
```javascript
updateMarkerPosition(event) {
      this.map.center = [event.latlng.lat, event.latlng.lng];
      this.map.markerLatLng = [event.latlng.lat, event.latlng.lng];
}
```
